cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunNativeImageTestTask.java

```diff
@@ -154,6 +156,12 @@ public RunNativeImageTestTask(PrintStream out, boolean rerunTests, String groupL
             singleExecTests = testList;
         }
         this.listGroups = listGroups;
+
+        if (workers <= 0) {
+            testWorkers = DEFAULT_TEST_WORKERS;
```

Comment:
imo, it is better to give an error or some kind of info for scenarios that `workers<1`

Reply:
We do have warning https://github.com/ballerina-platform/ballerina-lang/pull/40279/files#diff-1981767df1388fb1ab20720fba97d0ed0c37f6b09c856d2819254d3cc3768503R285